float64 abort_alt
float64 precision_land_mode
float64 yaw_angle
float64 lat
float64 lon
float64 altitude